### PR TITLE
Fixing lint before commit using lint-staged before commit

### PR DESCRIPTION
**Pull request by ritikrana4**
Fixes lint of the staged code using lint-staged before committing.

Issue: #194

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,8 +48,13 @@
...
   "husky": {
     "hooks": {
-      "pre-commit": "npm run hooks:pre-commit",
+      "pre-commit": "lint-staged",
       "pre-push": "npm run hooks:pre-push"
     }
+  },
+  "lint-staged": {
+    "./**/*.{js,jsx,ts,tsx,scss}": [
+      "npm run lint-fix"
```
Comment (sumitd94) [inline package.json:57]: Do not fix it, check for it.

File: package.json [hunk 2]
```diff
@@ -48,8 +48,13 @@
...
+    ]
   }
-}
+}
```
Comment (sumitd94) [inline package.json:60]: an empty line after this.

**Review by sumitd94 — CHANGES_REQUESTED** [review]

**Comment by ritikrana4** [comment]
Now will only check for lint issue.

**Comment by ritikrana4** [comment]
@sumitd94 Please review it.

**Comment by sumitd94** [comment]
#241

**Pull request closed without merging**